index.js

```diff
@@ -1,12 +1,28 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const dbinitRoute = require('./src/routes/dbinitRoute');
 
-const express = require('express')
-const app = express()
-const port = 3000
+const app = express();
+const port = 3000;
+const dbUrl = 'mongodb+srv://admin:admin@cluster0.jg02e.mongodb.net/myFirstDatabase?retryWrites=true&w=majority';
 
-app.get('/', (req, res) => {
-  res.send('Hello World!')
-})
+app.use(dbinitRoute);
 
-app.listen(port, () => {
-  console.log(`Example app listening at http://localhost:${port}`)
-})
\ No newline at end of file
+async function start() {
+  try {
+    await mongoose.connect(
```

Comment:
it's better to create a function `runDatabase` in a `config/db.js` file, to export it from there and to run it here.